maintainers/maintainer-list.nix

```diff
@@ -11980,4 +11980,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  nitsky = {
+    name = "nitsky";
+    email = "492793+nitsky@users.noreply.github.com";
+    github = "nitsky";
+    githubId = 492793;
+  };
```

Comment:
We try to keep this file alphabetically sorted - mind putting this in the middle?

Also, traditionally this would just go in a separate commit in the same PR where you added the package (i.e. as the first commit in PR #128889) rather than as a separate PR. But since this PR is here, it's probably fine.

Reply:
I added this as a commit to #128889 in sorted order. Closing this. Thanks!